0_global_functions_st.R

```diff
@@ -180,3 +180,30 @@ check_valid_cfg <- function(cfg, expected_no_args = 3) {
   invisible(cfg)
 }
 
+# create paths to stress test masterdata files
+create_stressdata_masterdata_file_paths <- function(data_prep_timestamp, twodii_internal) {
```

Comment:
in general: why not put this into the `R/` directory?

Reply:
True, how about adding it to utils.R?

Reply:
seems like a good place